pom.xml

```diff
@@ -402,7 +403,14 @@ are essentially embedding a slightly modified version of felix -->
                 <plugin>
                     <groupId>org.jacoco</groupId>
                     <artifactId>jacoco-maven-plugin</artifactId>
-                    <version>0.8.0</version>
+                    <version>${maven-jacoco-plugin.version}</version>
+		    <dependencies>
```

Comment:
Are you missing a `</plugin>` tag right here?

Reply:
No I am not. The plugin tag is below at line 826